cmd/lifecycle/builder.go

```diff
@@ -76,16 +76,17 @@ func (b *buildCmd) Exec() error {
 
 func (ba buildArgs) build(group lifecycle.BuildpackGroup, plan lifecycle.BuildPlan) error {
 	builder := &lifecycle.Builder{
-		AppDir:        ba.appDir,
-		LayersDir:     ba.layersDir,
-		PlatformDir:   ba.platformDir,
-		BuildpacksDir: ba.buildpacksDir,
-		PlatformAPI:   api.MustParse(ba.platformAPI),
-		Env:           env.NewBuildEnv(os.Environ()),
-		Group:         group,
-		Plan:          plan,
-		Out:           cmd.Stdout,
-		Err:           cmd.Stderr,
+		AppDir:          ba.appDir,
+		LayersDir:       ba.layersDir,
+		PlatformDir:     ba.platformDir,
+		BuildpacksDir:   ba.buildpacksDir,
+		PlatformAPI:     api.MustParse(ba.platformAPI),
+		Env:             env.NewBuildEnv(os.Environ()),
+		Group:           group,
+		Plan:            plan,
+		Out:             cmd.Stdout,
+		Err:             cmd.Stderr,
+		BuildpackFinder: &lifecycle.DefaultBuildpackFinder{},
```

Comment:
I would love to see `ba.buildpacksDir` be put into `lifecycle.DefaultBuildpackFinder{}` here to simplify the signatures of the find function. I think Emily mentioned the same thing.